lms/templates/emails/password_reset_subject.txt

```diff
@@ -0,0 +1,4 @@
+{% load i18n %}
```

Comment:
Minor change, I think it's better to add this to the path below instead. Similar to what [Django does](https://github.com/django/django/blob/1.8/django/contrib/auth/templates/registration/password_reset_subject.txt): 

 - `registration/password_reset_subject.txt`